src/components/core/filter-bar/filter-bar.tsx

```diff
@@ -174,7 +174,12 @@ export const FilterBar = ({ className, onChange, filters, defaultState = {} }: F
   ];
 
   return (
-    <div className={cx(className, "ctw-relative ctw-flex ctw-items-center ctw-space-x-2")}>
+    <div
+      className={cx(
+        className,
+        "ctw-items-left ctw-relative ctw-flex ctw-flex-col ctw-flex-wrap md:ctw-flex-row md:ctw-items-center"
```

Comment:
change to `sm` breakpoint
```suggestion
        "ctw-items-left ctw-relative ctw-flex ctw-flex-col ctw-flex-wrap sm:ctw-flex-row sm:ctw-items-center"
```